Target-MVD-Mathias/ViewControllers/SignUpViewController.swift

```diff
@@ -200,29 +204,17 @@ extension SignUpViewController: UITextFieldDelegate {
   func textFieldDidEndEditing(_ textField: UITextField) {
     switch textField {
       case nameTextField:
-        if nameIsInvalid() {
-          showErrorInForm(textField: nameTextField, errorLabel: nameErrorLabel)
-        } else {
-          hideErrorInForm(textField: nameTextField, errorLabel: nameErrorLabel)
-        }
+        toggleErrorInForm(error: nameIsInvalid(), textField: nameTextField, errorLabel: nameErrorLabel)
       case emailTextField:
-        if emailIsInvalid() {
-          showErrorInForm(textField: emailTextField, errorLabel: emailErrorLabel)
-        } else {
-          hideErrorInForm(textField: emailTextField, errorLabel: emailErrorLabel)
-        }
+        toggleErrorInForm(error: emailIsInvalid(), textField: emailTextField, errorLabel: emailErrorLabel)
       case passwordTextField, confirmPasswordTextField:
         checkPasswordErrors()
         
         if textField == passwordTextField {
```

Comment:
You could simplify this by doing: `firstTimeEditingPassword = textField != passwordTextField` instead of
```
if textField == passwordTextField {
  firstTimeEditingPassword = false
}
```